components/RecipeList.js

```diff
@@ -0,0 +1,52 @@
+import data from "@/db/data";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function RecipeList() {
+  if (!data) {
+    return <p>Data not found</p>;
+  }
+
+  return (
+    <>
+      <RecipeStyledUl>
+        {data.map((data) => (
+          <li key={data.name}>
+            <RecipeContainer>
+              <Image
+                src={data.imageURL}
+                alt={data.name}
+                width={150}
+                height={150}
+              />
+              <RecipeTitle>{data.name}</RecipeTitle>
+            </RecipeContainer>
+          </li>
+        ))}
```

Comment:
I think using `data` inside of the map could lead to confusion (especially as the variable for your dummy data is also called `data`). I would consider something like `recipe` if this is an array of recipes.
```suggestion
        {data.map((recipe) => (
          <li key={recipe.name}>
            <RecipeContainer>
              <Image
                src={recipe.imageURL}
                alt={recipe.name}
                width={150}
                height={150}
              />
              <RecipeTitle>{recipe.name}</RecipeTitle>
            </RecipeContainer>
          </li>
        ))}
```